requirements/tools/check_changes.py

```diff
@@ -25,11 +25,17 @@ def printing_table(columns: List[str]):
 
 
 def dump_changes(filename: Path):
-    subprocess.run(f"git --no-pager diff > {filename}", shell=True, check=True)
+    if filename.exists():
+        filename.unlink()
+    subprocess.run(
+        f"git --no-pager diff master..$(git rev-parse --abbrev-ref HEAD) > {filename}",
+        shell=True,
+        check=True,
+    )
 
 
 def tag_upgrade(from_version: Version, to_version: Version):
-    assert from_version < to_version
+    #assert from_version < to_version, (from_version, to_version)
```

Comment:
uncomment this :-)

Reply:
mad it even better, you now get a warning text if you downgrade accidentally